code/modules/medical/genetics/portagene.dm

```diff
@@ -107,7 +170,7 @@
 	power_change()
 		return
 
-	verb/eject()
+	verb/eject(var/mob/usr)
```

Comment:
This is going to cause you problems. Have you tested if the rightclick eject verb works like this?